src/cliffordsynthesis/encoding/SingleGateEncoder.cpp

```diff
@@ -119,6 +155,33 @@ LogicTerm SingleGateEncoder::createNoChange(
   return changes;
 }
 
+LogicTerm SingleGateEncoder::createNoChangeOnQubit(const std::size_t pos,
+                                                   const std::size_t q) {
+  auto noChange = LogicTerm(true);
+  noChange      = noChange && (tvars->x[pos + 1][q] == tvars->x[pos][q]);
+  noChange      = noChange && (tvars->z[pos + 1][q] == tvars->z[pos][q]);
+  return noChange;
```

Comment:
```suggestion
  return (tvars->x[pos + 1][q] == tvars->x[pos][q]) && (tvars->z[pos + 1][q] == tvars->z[pos][q]);
```
Should do the same thing and be much simpler, right?

Reply:
The function is not necessary anymore anyway. Removed it entirely.